src/VirtualMonoRepo/source-mappings.json

```diff
@@ -210,11 +215,23 @@
         },
         {
             "name": "winforms",
-            "defaultRemote": "https://github.com/dotnet/winforms"
+            "defaultRemote": "https://github.com/dotnet/winforms",
+            "include": [
+                "src/System.Windows.Forms.Design/src/Resources/colordlg.data",
+                "src/System.Windows.Forms/src/Resources/System/Windows/Forms/Design/Thumbs.db"
+            ],
+            "exclude": [
+                // Non-OSS license - https://github.com/dotnet/winforms/issues/10779
+                "https://github.com/dotnet/winforms/blob/main/src/System.Windows.Forms/tests/IntegrationTests/DesignSurface/LICENSE.txt"
```

Comment:
This is unrelated to the context of binary files. We can't just exclude the license and take the source since that's a violation of the license.

Reply:
That was a misunderstanding on my part. Working to resolve this now.

Reply:
@mthalman - I've cloaked the offending files in https://github.com/dotnet/installer/pull/18563/commits/d19dabb2cfbf7416fd367af9b90f52e8af31ccc2